src/backend/blockchain/src/routes.py

```diff
@@ -48,41 +50,65 @@ def proof():
     blockchain.append_block_to_chain(new_block, proof)
 
     blockchain.subtract_from_wallet(new_transaction.from_address, new_transaction.amount)
```

Comment:
We should move this subtract and add to wallet-api at some point. I'll leave it up to you to either add a TODO comment here and create an issue for it or do the refactoring now

Reply:
Thanks for pointing that out,  I will just change it with this PR